### PR TITLE
Enable thrust-math on windows

### PR DESCRIPTION
**Pull request by wenhuiNi**

### CODE REVIEW

**Review by yihwang-nv — APPROVED** [review]

File: features/feature_case/thrust/thrust-math.cu [hunk 1]
```diff
@@ -9,85 +9,217 @@
...
+  else{
+    std::cout<<"complex_pow1 "<<"test pass \n";
+  }
+  flag = flag & (std::abs(*hostRes-*Res)<1e-6);
+
+  // complex_pow2_kernel<<<1, 1>>>(deviceRes);
+  // cudaDeviceSynchronize();
+  cudaMemcpy(hostRes, deviceRes, sizeof(float), cudaMemcpyDeviceToHost);
+  cudaDeviceSynchronize();
+  // complex_pow2(Res);
```
Comment (yihwang-nv) [inline features/feature_case/thrust/thrust-math.cu:167]: Please remove these unused code.

Comment (wenhuiNi) [reply features/feature_case/thrust/thrust-math.cu:167]: not unused, it will be re-enabled after intel/llvm fix related API.
Leave a mark

**Review by tangjj11 — APPROVED** [review]

File: features/feature_case/thrust/thrust-math.cu [hunk 2]
```diff
@@ -9,85 +9,217 @@
...
+{
+  auto c = thrust::polar(11.48569f, 1.33698f);
   c = thrust::pow(1.0f, c);
-  c = thrust::pow(c, 1.0f);
+  *res = thrust::abs(c);  
+}
+
+// __global__ void complex_pow2_kernel(float * res)
+// {
+//   auto c = thrust::polar(5.48569f, 1.33698f);
```
Comment (ShengchenJ) [inline features/feature_case/thrust/thrust-math.cu:38]: Please remove the comment code?

File: features/feature_case/thrust/thrust-math.cu [hunk 3]
```diff
@@ -9,85 +9,217 @@
...
+{
+  auto c = thrust::polar(11.48569f, 1.33698f);
   c = thrust::pow(1.0f, c);
-  c = thrust::pow(c, 1.0f);
+  *res = thrust::abs(c);  
+}
+
+// void complex_pow2(float * res)
+// {
+//   auto c = thrust::polar(5.48569f, 1.33698f);
```
Comment (ShengchenJ) [inline features/feature_case/thrust/thrust-math.cu:88]: Same as above

File: features/feature_case/thrust/thrust-math.cu [hunk 4]
```diff
@@ -9,85 +9,217 @@
...
   float *deviceRes;
+  bool flag = true;
   cudaMalloc((float **)&deviceRes, sizeof(float));
-  complex_math_kernel<<<1, 1>>>(deviceRes);
+
+  // complex_log10_kernel<<<1, 1>>>(deviceRes);
+  // cudaDeviceSynchronize();
+  // cudaMemcpy(hostRes, deviceRes, sizeof(float), cudaMemcpyDeviceToHost);
+  // cudaDeviceSynchronize();
+  // complex_log10(Res);
```
Comment (ShengchenJ) [inline features/feature_case/thrust/thrust-math.cu:125]: Same

File: features/feature_case/thrust/thrust-math2.cu [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+}
+
+__global__ void complex_atanh_kernel(float * res)
+{
+  auto c = thrust::polar(11.48569f, 1.33698f);
+  c = thrust::atanh(c);
+  *res = thrust::abs(c);  
+}
+
+void complex_atanh(float * res)
```
Comment (ShengchenJ) [inline features/feature_case/thrust/thrust-math2.cu:48]: Could you merge the host code and device code to a function call?

**Review by ShengchenJ — APPROVED** [review]

**Pull request merged**